synapse/util/async.py

```diff
@@ -32,20 +32,15 @@
 logger = logging.getLogger(__name__)
 
 
-@defer.inlineCallbacks
-def sleep(seconds):
```

Comment:
I think you've missed a call to this in `background_updates.py`.

Reply:
fix'd